pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -11171,7 +11171,7 @@
       a: "The Hour of Code started as a one-hour introduction to computer science, designed to demystify \"code\", to show that anybody can learn the basics, and to broaden participation in the field of computer science. It has since become a worldwide effort to celebrate computer science, starting with 1-hour coding activities but expanding to all sorts of community efforts. Check out the [tutorials and activities](%{tutorials_url})."
     when_is_hoc:
       q: "When is the Hour of Code?"
-      a: "The Hour of Code takes place each year during Computer Science Education Week. The %{campaign_date_year} Computer Science Education Week will be %{campaign_date_full}, but you can host an Hour of Code event all year-round. Computer Science Education Week is held annually in recognition of the birthday of computing pioneer Admiral Grace Murray Hopper (December 9, 1906)."
+      a: "The Hour of Code takes place each year during [Computer Science Education Week](%{csedweek_url}). The %{campaign_date_year} Computer Science Education Week will be %{campaign_date_full}, but you can host an Hour of Code event all year-round. Computer Science Education Week is held annually in recognition of the birthday of computing pioneer [Admiral Grace Murray Hopper](%{hopper_url}) (December 9, 1906)."
```

Comment:
Won't we need a new string here? Any existing translations won't have the right interpolations...

Reply:
jk I tested locally and the hyperlinks just don't appear so this might be alright!

Reply:
Oh good point. I figured since these strings aren't used anywhere on the site yet it would be ok, but I'll make new strings just in case.